iio/iio_app/iio_app.c

```diff
@@ -106,7 +106,7 @@ int32_t iio_app(struct iio_app_desc *desc)
 	int32_t status;
 
 	while(1) {
-		status = tinyiiod_read_command(desc->iiod);
+		status = iio_step(desc->iio);
```

Comment:
This can be renamed to `iio_read_command`, maybe.

Reply:
I am not sure how to name it.  iio_read_command makes me think that I should expect some output from the function, that's way I though it is not a good idea to name it this way.